src/component/card/ProductCard.js

```diff
@@ -24,7 +24,17 @@ const ProductCard = ({ productDetails }) => {
       <div className="card">
         <div className="card-image-container">
           <img className="image-responsive" src={Image} alt={alt} />
-          <i className="card-badge far fa-heart"></i>
+          {state.wishlist.find((product) => product._id === _id) ? (
```

Comment:
instead of doing find directly here, you can create a variable above and use that variable here. It will make code a little cleaner

Reply:
Thanks for suggestion will try to implement that